src/index.ts

```diff
@@ -1,10 +1,18 @@
-import { Users } from './services/users';
-import { Logger } from './services/logger';
+import {Logger} from './services/logger';
 
-import type { User, ApiConfig } from './types';
+import {createIoCContainer, IoCResources} from "./ioc";
 
-const renderUsers = async (config: ApiConfig) => {
-  const usersService = new Users(config);
+import type {User} from './types';
+import IoCContainer from "ioc-lite";
+
+const renderUsers = async (ioc: IoCContainer<IoCResources>) => {
+
+    // This is somehow working without resolving the logger and http,
+    // I failed to find the true reason of it and will be very grateful to you if you can explain it to me
+    // ioc.resolve('logger');
+    // ioc.resolve('http');
```

Comment:
I'm not sure I understand. What do you mean by "This is somehow working..." ?

Reply:
done